andela/wg-aces
# #159250138 add actual meal eaten

### What does this PR do?
Upgrade the nutrition diary feature

### Description of Task to be completed?
The nutritional plan diary should be upgraded for users to be able to enter the food he/she actually ate during the day, in addition to what was planned with the nutritional plan.

### How should this be manually tested?
Login as admin
Navigate to ```Nutrition```
Navigate to ```Nutrition plans```
Navigate to ```Add nutrition plan```
 Click ```Add new meal``` and fill in the form to create a planned meal
Hoover over the meal item and a button ```Actual meal``` will appear in the first column.
Click ```Add actual meal``` and fill the form
View the actual meal item clicking the tab ```Actual meal```
View the nutritional value clicking ```nutritional value``` 

### Any background context you want to provide?
Currently, a user can enter into the nutrition plan the food they plan to eat. However, sometimes things do not go as planned and users end up eating something else, but they need to keep track of their diet.

### What are the relevant pivotal tracker stories?
[#159250138](https://www.pivotaltracker.com/story/show/159250144)

### Screenshots
![image](https://user-images.githubusercontent.com/30747298/44212418-02cbf780-a174-11e8-8938-3365a7483a09.png)

![image](https://user-images.githubusercontent.com/30747298/44212459-1b3c1200-a174-11e8-804a-d5b7288a8820.png)
